maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/service/ValidationService.java

```diff
@@ -32,8 +32,10 @@
 @Service
 @RequiredArgsConstructor
 public class ValidationService {
-    public static final String CANNOT_MODIFY_APPLICATION_ERROR =
-            "Application has been modified by another user";
+    public static final String CANNOT_MODIFY_APPLICATION_ERROR = """
+            Application has been modified by another user. Please try logging out and logging back in again.
+            Please contact your system administrator if this issue persists.
+            """;
```

Comment:
I would expect adding a call to action to error messages or making them more user friendly would be something we do in the frontend, though I guess this isn't possible in this case?

Reply:
This is a good suggestion. We can identify the list of error messages and add a generic amendment in the MAAT application layer, I will create a story for this.